client/src/components/App.jsx

```diff
@@ -1,20 +1,70 @@
 import React from 'react';
+import styled from 'styled-components';
+import Modal from './Modal.jsx';
+import LargeImage from './MainLargeImage.jsx';
+import SmallImages from './MainSmallImages.jsx';
+import $ from 'jquery';
 import { dummyData } from '../../../data';
 
+const Parent = styled.div`
+  display: flex;
+  width: 100%;
+  height: 430px;
+`;
+
+const Button = styled.button`
+  position: absolute;
+  top: 390px;
+  right: 24px;
+  padding: 6px 15px 6px 15px;
+  color: #484848;
+  border-radius: 4px;
+  font-size: 14px;
+  cursor: pointer;
+`;
+
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       data: dummyData[0],
+      show: false,
     };
+    this.showModal = this.showModal.bind(this);
+  }
+
+  componentDidMount() {
+    $.ajax({
+      url: '/api/photos/:offering',
```

Comment:
Curious - does the URL work in this case? Per my understanding, `/:offering` is only used in server route setup logic to template an expected value to be passed into the route. I would expect the url here to be something like `/api/photos/4`, but correct me if I'm wrong!

If the AJAX call is working, I might expect all photos to be sent back from the API since no specific offering id is being passed with the URL.